### PR TITLE
Updated SQLStockLevelsConfig.overstock_threshold to accept 5 digits

### PR DESCRIPTION
**Pull request by orangejenny**
## Summary
Followup for https://github.com/dimagi/commcare-hq/pull/27597/ - found a document on prod that needs a bigger column.

## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### Automated test coverage

There are tests for this migration.

### QA Plan

Minor schema change, not requesting QA.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

File: corehq/apps/commtrack/models.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ class SQLStockLevelsConfig(models.Model):
     # All of these are in months
     emergency_level = models.DecimalField(default=0.5, max_digits=4, decimal_places=2)
     understock_threshold = models.DecimalField(default=1.5, max_digits=4, decimal_places=2)
```
Comment (mjriley) [inline corehq/apps/commtrack/models.py:191]: should understock also be updated to 5? Seems like the two fields are related?

**Review by mjriley — APPROVED** [review]

**Comment by dannyroberts** [comment]
@orangejenny is this a currently unused table and that's why it's safe to do without thinking harder about the migration backwards compatibility and related issues?

File: corehq/apps/commtrack/models.py (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/commtrack/models.py:191]: I'll find out when I run the migration again (arguably I could check beforehand, just trying not to spend much time on this PR). I don't have great context on the usage of these fields, but for what it's worth, the project that triggered the overflow has an overstock threshold of 500 but an understock threshold of 10, so they don't necessarily mirror each other.

**Comment by orangejenny** [comment]
@dannyroberts I'm not expecting backwards compatibly to be an issue because this is a new table for the couch to sql migration of commtrack config docs, and because enlarging a column ought to be backwards compatible with existing data. This is a small (< 1000 rows) table that isn't currently being read from.

**Review by dannyroberts — APPROVED** [review]

**Comment by orangejenny** [comment]
Deployed to india without incident.

**Pull request merged**